sdk/core/azure_core_test_macros/src/test.rs

```diff
@@ -140,6 +149,13 @@ impl Parse for Attributes {
             }
         }
 
+        if attrs.live && attrs.playback {
```

Comment:
Why not? If setup is hard - as it is now for this - but could work live and work only off a recording in CIs, why couldn't it be both? I'd say: allow it, and if it fails in one or the other, let them fix it and change the attributes as needed.